crm_location/README.rst

```diff
@@ -0,0 +1,13 @@
+CRM location
+============
+
+This module introduces a better zip.
+
+It enables zip, city, state and country auto-completion on lead and companies.
+
+Credits
+=======
+
+Contributors
+------------
+* Endika Iglesias <endikaig@antiun.com>
```

Comment:
Add bug tracker section recently added to the template: https://github.com/OCA/maintainer-tools/tree/master/template/module#bug-tracker